src/app/meetups/add-meetups/meetups-add.component.html

```diff
@@ -10,9 +10,6 @@
     <form class="form-spacing" [formGroup]="meetupForm" novalidate>
       <mat-form-field>
         <input matInput i18n-placeholder placeholder="Title" formControlName="title" required>
-        <mat-error>
```

Comment:
Because there will still be a validator, you should put the `mat-error` element back in.